go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
Yeah, actually this was added when the codegangsta/cli returned method had a different signature(`func(c *cli.Context)` instead of `func(c *cli.Context) error`) and at that times this was quite valid.